setpass/config.py

```diff
@@ -45,7 +45,28 @@
 
     cfg.IntOpt('max_attempts',
                default=3,
-               help='Max number of pin attempts before lockout.')
+               help='Max number of pin attempts before lockout.'),
+
+    cfg.StrOpt('helpdesk_email',
+               default='root@localhost',
+               help='Destination email for password reset notification.'),
+
+    cfg.StrOpt('ticket_sender',
+               default='root@localhost',
+               help='Source email for password reset notification.'),
+
+    cfg.StrOpt('ticket_subject',
+               default='New Ticket',
+               help='Email address for password reset notification.'),
+
+    cfg.StrOpt('mail_ip',
```

Comment:
Thanks, I didn't realize that would be an option, I haven't worked closely with oslo before.  I will look into it further.